andela/valinor-ah-backend
# #161291024 Build out the report article feature

#### What does this PR do?

Have an endpoint to report articles

#### Description of Task to be completed?

- add tests
- add route
- add route handler
- add route authentication and validation

#### How should this be manually tested?

After cloning the repo, cd into the folder and run `npm install, then check that the endpoint works as follows:
- Open postman
- Select **Post**
- Type in the route
- Fill out the body of the request
- Press enter

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

[#161291024]()

#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-11-15 at 7 57 36 pm" src="https://user-images.githubusercontent.com/15168965/48574912-bf1e9080-e910-11e8-93a3-c091118f18df.png">


#### Questions:

N/A
